lib/game_teams.rb

```diff
@@ -0,0 +1,24 @@
+class GameTeam
+
+  attr_reader :game_id,
+              :team_id,
+              :hoa,
+              :result,
+              :settled_in,
+              :head_coach,
+              :goals,
+              :shots,
+              :tackles
+
+  def initialize(game_team_params)
```

Comment:
This looks good, we can double check we have initialized all the ones we need for the data next steps.